### DISCUSSION TITLE
v1.9.3

### DISCUSSION THREAD

**Discussion by n4ze3m** [n4ze3m/dialoqbase#286] [Announcements]
## What's Changed

- [NEW] Ability to increase the file upload limit
- Added pagination to the data-source page
- [NEW] OpenAI-compatible chat endpoint api

## Blog

- [Creating a Tiny AI App Using Dialoqbase API](https://dialoqbase.n4ze3m.com/reference/examples/creating-tiny-ai-app-using-dialoqbase-api.html)

**Full Changelog**: https://github.com/n4ze3m/dialoqbase/compare/v1.9.2...v1.9.3

<hr /><em>This discussion was created from the release <a href='https://github.com/n4ze3m/dialoqbase/releases/tag/v1.9.3'>v1.9.3</a>.</em>

**Comment by edwhiskey** [comment]
I love your work. However, I am seeking to have websearch and live data features for Dialoqbase.

**Reply by n4ze3m** [reply]
Hey, first of all, thank you for sponsoring <3. And yes, web search can be included in upcoming updates. Also, what do you mean by live data? I didn't get it
